source/partials/_footer.erb

```diff
@@ -1,3 +1,11 @@
-<footer class="nav-footer">
+<footer class="footer">
+  <nav class="nav">
+    <ul class="social-list">
+      <li class="social twitter"><a href="http://twitter.com/plibmttbhgaty">Twitter</a></li>
+      <li class="social github"><a href="http://github.com/plibmttbhgaty">Github</a></li>
+      <li class="social email"><a href="mailto:star+plibmttbhgaty@mit.edu">Email</a></li>
+    </ul>
+  </nav>
+
   <p>&copy; <%= Date.today.year %> PLIBMTTBHGATY</p>
```

Comment:
Maybe add a "designed [with love] by thoughtbot" or "designed by Connie Chan" because you've been spending investment time on this? If we don't want to add that here, maybe we can mention this design work on the "Products & Open Source" section of the new thoughtbot website.